app/serializers/api/v1/organisation_names_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module Api::V1
+  class OrganisationNamesSerializer < ApplicationSerializer
+    embed :ids, include: true
+
+    attributes :id, :name_en, :name_zh_tw
```

Comment:
Do we need to send both the English AND the Chinese name each time? Would it be better to just send the name of the organisation in the language that is required? e.g. see https://github.com/crossroads/api.goodcity/blob/master/app/serializers/api/v1/territory_serializer.rb for an example.

Are there cases where we have the chinese name but not the english name? Or vice-versa? In which case we could use COALESCE to ensure we have some values

Reply:
Thats a good suggestion, I used the existing component from Stock which had both english and chinese name displayed at a time in searched list. So, created that here and displayed both English and Chinese names. Will add your suggested change.

Reply:
On changing name on the basis of English and Chinese will affect the Organisation model on frontend because it sends both English and Chinese names and managing conditional names in model will require lots of changes. I feel its okay to send both Chinese and english names from api.

Let me know your thoughts on this.

Thanks.